docs/source/Compiler.md

```diff
@@ -116,6 +116,9 @@ Additional options:
 
 -   `--gen-generated` : Add @Generated annotation for Java.
 
+-   `--kotlin-gen-jvmstatic` : Add @JvmStatic annotation for Kotlin methods
```

Comment:
lets call this just `--gen-jvmstatic` as all the other options above don't have the language name in them either.

Reply:
The option name changed to --gen-jvmstatic